src/commands/Utility/language.ts

```diff
@@ -148,6 +153,28 @@ const command: Command = {
 					})
 				}
 			})
+		} else if (subCommand === "reset") {
+			const result = await collection.updateOne({ id: interaction.user.id }, { $set: { lang: null } })
```

Comment:
Wouldn't this be better?
```suggestion
			const result = await collection.updateOne({ id: interaction.user.id }, { $unset: { lang: true } })
```

Reply:
Resolved in recent commit.